addons/mail/static/src/models/discuss_view.js

```diff
@@ -149,6 +110,17 @@ registerModel({
             }
             return clear();
         },
+        /**
+         * @private
+         */
+        _onDiscussThreadChanged() {
+            if (this.discuss.discussView) {
```

Comment:
condition is not necessary, we are already on the discussView itself so it will not execute if it is gone